Mathlib/MeasureTheory/Constructions/Prod/Integral.lean

```diff
@@ -504,15 +509,20 @@ theorem set_integral_prod (f : α × β → E) {s : Set α} {t : Set β}
   exact integral_prod f hf
 #align measure_theory.set_integral_prod MeasureTheory.set_integral_prod
 
-theorem integral_prod_mul {L : Type*} [IsROrC L] (f : α → L) (g : β → L) :
-    ∫ z, f z.1 * g z.2 ∂μ.prod ν = (∫ x, f x ∂μ) * ∫ y, g y ∂ν := by
-  by_cases h : Integrable (fun z : α × β => f z.1 * g z.2) (μ.prod ν)
+theorem integral_prod_smul {𝕜 : Type*} [IsROrC 𝕜] [NormedSpace 𝕜 E] (f : α → 𝕜) (g : β → E) :
```

Comment:
In `Integrable.prod_smul`, you generalized `IsROrC` to `NontriviallyNormedField`. Here you kept `IsROrC`. Is the generalization impossible here?